paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -55,10 +56,11 @@ def add_subparser(subparsers):
             required=True,
         ).completer = lazy_choices_completer(list_instances)
         status_parser.add_argument(
-            '-c', '--cluster',
-            help='The PaaSTA cluster that has the service you want to %s. Like norcal-prod' % lower,
-            required=True,
-        ).completer = lazy_choices_completer(utils.list_clusters)
+            '-c', '--clusters',
```

Comment:
There is an [`nargs`](https://docs.python.org/2/library/argparse.html#nargs) option you can use to get a list.

Also, since we know the list of possible clusters, it might nice to start using the [choices](https://docs.python.org/2/library/argparse.html#choices) option.